### PR TITLE
Radiation storms no longer do random mutations in of itself, but are far more lethal

### PR DESCRIPTION
**Pull request by silicons**
## About The Pull Request

Radstorms are boring, let's make 'em more fun because gorilla monkies are great right?

## Why It's Good For The Game

Random mutations are honestly just annoying as shit, this makes them more of a threat but not by just mutating the hell out of you. If it kills too many afkers lockernapping we can address that.

## Changelog
:cl:
Experimental: Radiation storms no longer do random mutations directly, but do far more direct radiation.
/:cl:

### CODE REVIEW

**Comment by MrJWhit** [comment]
I know nothing about radiation code, will this apply the radiation dust, where you need to shower off, or will this just give the effects directly to the body?

**Comment by silicons** [comment]
@MrJWhit ah, just radiation level for mob, not contamination.
contamination would be cancer :joy:

**Comment by Ghommie** [comment]
Aaah, this brings me back when gorillas were first merged, and some evil genius released forty monkeys in the halls just before a radstorm event, good old days...
Good change since it kills afk personas instead of turning them into twitching, gasping, vulgar invisible annoyances.

File: code/datums/weather/weather_types/radiation_storm.dm [hunk 1]
```diff
@@ -22,27 +22,17 @@
...
-					H.randmuti()
-					if(prob(50))
-						if(prob(90))
-							H.randmutb()
-						else
-							H.randmutg()
-						H.domutcheck()
-		L.rad_act(20)
+	var/ratio = 1 - (min(resist, 100) / 100)
+	L.rad_act(radiation_intensity * percent)
```
Comment (Ghommie) [inline code/datums/weather/weather_types/radiation_storm.dm]:
```suggestion
	L.rad_act(radiation_intensity * ratio)
```
Bad webedit.

**Review by Putnam3145 — APPROVED** [review]
might wanna test this to see if it's too onerous

**Comment by Ghommie** [comment]
Putnam is going to nerf radiation anyway, so I don't mind insta-merging this.

**Pull request merged**